packages/core/workflows-sdk/src/helper/workflow-export.ts

```diff
@@ -140,6 +140,7 @@ function createContextualWorkflowRunner<
     }
 
     context.transactionId ??= ulid()
+    context.eventGroupId ??= ulid()
```

Comment:
should we consider using the same as transactionId?

Reply:
Not sure 🤔 we can see that later? Its on my todo 😉